packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -392,7 +394,21 @@ class FlutterDriver {
   final VMServiceClient serviceClient;
 
   /// JSON-RPC client useful for sending raw JSON requests.
-  final rpc.Peer _peer;
+  rpc.Peer _peer;
+
+  String _dartVmReconnectUrl;
+
+  Future<void> _restorePeerConnectionIfNeeded() async {
+    if (!_peer.isClosed || _dartVmReconnectUrl == null) {
+      return;
+    }
+
+    _log.warning('Peer connection is closed! Try to restore the connection...');
```

Comment:
Is this meant to be `Trying to restore the connection...`?

Reply:
Done.